### PR TITLE
Add Bane Of Wargs Website

### PR DESCRIPTION
**Pull request by OcelotWalrus**
hey!

### CODE REVIEW

**Review by lumbar527 — COMMENTED** [review]
I'd like to wait until you deploy https://bane-of-wargs.github.io.

**Comment by OcelotWalrus** [comment]
Yes.

**Comment by lumbar527** [comment]
Why'd you archive the Cromha-Expansion plugin?

File: bane-of-wargs.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <p>What is BANE OF WARGS?</p>
 	<p>Bane Of Wargs is a game engine that you can use to create any story, item, mount, weapon, village or anything you can imagine. It is free, and open-source, and you can fork the game engine and tweak as you would like, to make it like you'd like!</p>
 	<p>You have the ability to create mods/plugins that you can load in the main menu and the mod content will overwrite the vanilla content, which make you able to totally modify the story of the base game.</p>
-	<p>You can find the game repository at <a href="https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs" target="_blank">https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs</a></p>
+	<p>You can find the game repository at <a href="https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs" target="_blank">https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs</a> and its website at <a href="https://bane-of-wargs.github.io" target="_blank">https://bane-of-wargs.github.io</a>.</p>
```
Comment (OcelotWalrus) [inline bane-of-wargs.html]:
```suggestion
	<p>You can find the game repository at <a href="https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs" target="_blank">https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs</a> and its website at <a href="https://ocelotwalrus.github.io/bane-of-wargs.github.io/" target="_blank">https://ocelotwalrus.github.io/bane-of-wargs.github.io/</a>.</p>
```

**Comment by OcelotWalrus** [comment]
> Why'd you archive the Cromha-Expansion plugin?

Well, it's kinda outdated now, and I don't plan on working on it in the future. I just deployed the website at https://ocelotwalrus.github.io/bane-of-wargs.github.io.

**Comment by OcelotWalrus** [comment]
Thanks!

**Pull request merged**